app/controllers/api/v1/users_controller.rb

```diff
@@ -18,6 +18,16 @@ def show
         render json: UserSerializer.new(User.find(params[:id]))
       end
 
+      def update
+        user = User.find(params[:id])
+        user.assign_attributes(user_params)
```

Comment:
I want to learn more about where `assign_attributes` is coming from. It looks like its 
- updating `self`
- saving the change to self
Do I have that correct?